content/blog/2017/2017-02-06-declarative-pipeline-ga.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Declarative Pipeline Syntax 1.0 is GA"
+tags:
+- pipeline
+- blueocean
+author: hrmpw
+---
+[NOTE]
+====
+This is a guest post by
+link:https://github.com/HRMPW[Patrick Wolf],
+Director of Product Management at
+link:http://cloudbees.com[CloudBees]
+and contributor to
+the Jenkins project.
+====
+
+I am very excited to announce the addition of
+link:https://plugins.jenkins.io/pipeline-model-definition[Declarative Pipeline syntax]
+1.0 to
+link:https://plugins.jenkins.io/workflow-aggregator[Jenkins Pipeline.]
+We think this new syntax will enable everyone involved in DevOps, regardless of expertise,
+to participate in the continuous delivery process. Whether creating, editing or reviewing
+a pipeline, having a straightforward structure helps to understand and predict the
+flow of the pipeline and provides a common foundation across all pipelines.
+
+= Pipeline as Code
```

Comment:
I think we start at `==` because the title is provide by header.
== Pipeline as Code